frontera/contrib/backends/sqlalchemy/__init__.py

```diff
@@ -161,9 +166,15 @@ def _create_page(self, obj):
         db_page.fingerprint = obj.meta['fingerprint']
         db_page.state = Page.State.NOT_CRAWLED
         db_page.url = obj.url
-        db_page.depth = 0
         db_page.created_at = datetime.datetime.utcnow()
         db_page.meta = obj.meta
+
+        if not isinstance(obj, frontera_response):
```

Comment:
I don't want to overwrite the response headers and stuff on the request headers, because maybe we would want to retry this request later, I imagine the database to be only request store and not response and response store.